lib/unique_elements.rb

```diff
@@ -1,6 +1,25 @@
-# This method takes an array of elements (these might be strings, integers, floats, or a
-# combination of the above), and returns an array of the elements that appear once and only
-# once.
+# For each value in the array that has a match, remove that value.
+
 def find_unique_elements(arr)
-  
-end
+	has_match = 0
```

Comment:
Nice work. Defining `has_match` and `y` as equal to zero twice in the first 10 or so lines of your code seems a little repetitive. What happens if you comment out lines 4 and 6?